modules/renter/files.go

```diff
@@ -147,21 +148,46 @@ func (f *file) redundancy(isOffline func(types.FileContractID) bool) float64 {
 		return -1
 	}
 	for _, fc := range f.contracts {
+		offline, gfr := contractStatus(fc.ID)
```

Comment:
gfr is not readable to someone unfamiliar with the code

Reply:
done